packages/graphql-codegen-golang/src/generator.ts

```diff
@@ -267,7 +267,8 @@ export class GolangGenerator {
         nonNull
       )
     }
-    throw new Error(`field type "${type}" not supported!`)
+    console.debug(`Skipped unsupported field type "${type.name.value}"`)
+    return ''
```

Comment:
@ctison – first, thanks for merging and releasing new version.

There's a problem with only returning `''` here, because generated code will include type name without type information, which will make generated file invalid Go syntax. In other words, `l.push` above shouldn't happen if type isn't supported like I did [here](https://github.com/ctison/graphql-codegen-golang/pull/201/commits/92fddfa13a6cfab2b988ebf2dfe107374c7ebbdf#diff-2e4a40a086feebe32628af77307427a8b50fefb741f48c881319a1a39844682aR220).